base_delivery_carrier_label/models/delivery_carrier.py

```diff
@@ -16,7 +16,7 @@ class DeliveryCarrier(models.Model):
     )
 
     def alternative_send_shipping(self, pickings):
-        return [{"exact_price": False, "tracking_number": False}]
+        return {}
```

Comment:
@fshah-initos Why removing this ?

Reply:
@rousseldenis I've removed it for testing purposes now I added it.